x/erc20/keeper/msg_server.go

```diff
@@ -413,113 +287,6 @@ func (k Keeper) convertERC20NativeToken(
 	return &types.MsgConvertERC20Response{}, nil
 }
 
-// convertCoinNativeERC20 handles the coin conversion for a native ERC20 token
-// pair:
-//   - escrow Coins on module account
-//   - unescrow Tokens that have been previously escrowed with ConvertERC20 and send to receiver
-//   - burn escrowed Coins
-//   - check if token balance increased by amount
-//   - check for unexpected `Approval` event in logs
-func (k Keeper) convertCoinNativeERC20(
-	ctx sdk.Context,
-	pair types.TokenPair,
-	msg *types.MsgConvertCoin,
-	receiver common.Address,
-	sender sdk.AccAddress,
-) (*types.MsgConvertCoinResponse, error) {
```

Comment:
we need to keep this for IBC coins -> ERC-20 conversions

Reply:
Ok but it needs to be refactored to now use `MsgConvertCoin` ?